packages/typedefs/src/interaction.ts

```diff
@@ -0,0 +1,85 @@
+
+export const UNKNOWN /*                       */ = "UNKNOWN"
+export const SCRIPT /*                        */ = "SCRIPT"
+export const TRANSACTION /*                   */ = "TRANSACTION"
+export const GET_TRANSACTION_STATUS /*        */ = "GET_TRANSACTION_STATUS"
+export const GET_ACCOUNT /*                   */ = "GET_ACCOUNT"
+export const GET_EVENTS /*                    */ = "GET_EVENTS"
+export const PING /*                          */ = "PING"
+export const GET_TRANSACTION /*               */ = "GET_TRANSACTION"
+export const GET_BLOCK /*                     */ = "GET_BLOCK"
+export const GET_BLOCK_HEADER /*              */ = "GET_BLOCK_HEADER"
+export const GET_COLLECTION /*                */ = "GET_COLLECTION"
+export const GET_NETWORK_PARAMETERS /*        */ = "GET_NETWORK_PARAMETERS"
```

Comment:
I plan to address it in a separate pr, as it requires changes that could change the logic

Reply:
Totally understand the reasoning - however, this would be a breaking change if we changed this later, so I'd say it's probably better if we settle on the final implementation now as oppose to attempting to migrate later.  I know that these constants are essentially internal, but I'd prefer to adhere to semver guidelines.

I don't think there's any logic change if you use as string-based enum, anyways.  It's effectively just namespaced constants so you should just need to replace references to these variables.

i.e.
```ts
enum InteractionType {
    Transaction = "TRANSACTION"
}
```

and replace existing references pointing at `TRANSACTION` to `InteractionType.Transaction`